Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerOsInformation.ps1

```diff
@@ -276,8 +276,9 @@ function Invoke-AnalyzerOsInformation {
         Add-AnalyzedResultInformation @params
     }
 
-    if ($osInformation.NetworkInformation.HttpProxy.ProxyAddress -ne "None" -and
-        $osInformation.NetworkInformation.HttpProxy.ProxyAddress -ne $exchangeInformation.GetExchangeServer.InternetWebProxy) {
+    if (($osInformation.NetworkInformation.HttpProxy.ProxyAddress -ne "None") -and
+        ($exchangeInformation.BuildInformation.ServerRole -ne [HealthChecker.ExchangeServerRole]::Edge) -and
+        ($osInformation.NetworkInformation.HttpProxy.ProxyAddress -ne $exchangeInformation.GetExchangeServer.InternetWebProxy.Authority)) {
```

Comment:
Does this work when from a tools box that is on a different build of Exchange? 

Like when `Get-ExchangeServer` is run, the `AdminDisplayVersion` is a `string` type not an object type.